api/src/main/java/com/cloud/host/Host.java

```diff
@@ -57,6 +57,10 @@ public static String[] toStrings(Host.Type... types) {
     public static final String HOST_VOLUME_ENCRYPTION = "host.volume.encryption";
     public static final String HOST_INSTANCE_CONVERSION = "host.instance.conversion";
```

Comment:
@nvazquez this details would be redundant when virt-v2v version is added in the details. so, can you remove this detail in upgrade path or when 'host.virtv2v.version' details is added?